CMakeLists.txt

```diff
@@ -147,107 +147,95 @@ if(ONNX_BUILD_TESTS)
 endif()
 
 if(NOT ONNX_BUILD_CUSTOM_PROTOBUF)
-  if((ONNX_USE_LITE_PROTO AND TARGET protobuf::libprotobuf-lite) OR ((NOT ONNX_USE_LITE_PROTO) AND TARGET protobuf::libprotobuf))
-    # Sometimes we need to use protoc compiled for host architecture while linking
-    # libprotobuf against target architecture. See https://github.com/caffe2/caffe
-    # 2/blob/96f35ad75480b25c1a23d6e9e97bccae9f7a7f9c/cmake/ProtoBuf.cmake#L92-L99
-    if(EXISTS "${ONNX_CUSTOM_PROTOC_EXECUTABLE}")
-      message(STATUS "Using custom protoc executable")
-      set(ONNX_PROTOC_EXECUTABLE ${ONNX_CUSTOM_PROTOC_EXECUTABLE})
-    else()
-      if(TARGET protobuf::protoc)
-        set(ONNX_PROTOC_EXECUTABLE $<TARGET_FILE:protobuf::protoc>)
-      endif()
-    endif()
-  else()
-    # Customized version of find Protobuf. We need to avoid situations mentioned
-    # in https://github.com/caffe2/caffe2/blob/b7d983f255ef5496474f1ea188edb5e0ac4
-    # 42761/cmake/ProtoBuf.cmake#L82-L92 The following section is stolen from
-    # cmake/ProtoBuf.cmake in Caffe2
-    find_program(Protobuf_PROTOC_EXECUTABLE
+  # Customized version of find Protobuf. We need to avoid situations mentioned
+  # in https://github.com/caffe2/caffe2/blob/b7d983f255ef5496474f1ea188edb5e0ac4
+  # 42761/cmake/ProtoBuf.cmake#L82-L92 The following section is stolen from
+  # cmake/ProtoBuf.cmake in Caffe2
```

Comment:
can you add a link here?

Reply:
What link? But this is the old comment and things have changed a lot since then. It could be removed...